### DISCUSSION TITLE
New Versioning for date based versions

### DISCUSSION THREAD

**Discussion by j2L4e** [renovatebot/renovate#22130] [Suggest an Idea]
### Type of discussion.

I'm proposing an idea

### Tell us more.

What would you like Renovate to be able to do?
We're struggling with "RELEASE.2023-05-04T18-10-16Z" style versioning as [mc](https://github.com/minio/mc/tags) uses. Parsing time-based versions doesn't seem possible at the moment. Having a versioning type that allows date parsing/comparison would be needed here.

If you have any ideas on how this should be implemented, please tell us here.
Making the date format configurable would be great, so you could use it like regex versioning:

date:'RELEASE.'yyyy-MM-dd'T'HH-mm-ssX
(see https://codesandbox.io/s/date-fns-playground-forked-kn3o14?file=/src/index.js)

**Comment by rarkins** [comment]
I think that https://github.com/renovatebot/renovate/issues/21759 would solve your problem. You don't need it to be date aware, just sorting properly

**Reply by j2L4e** [reply]
Assuming big-endian dates you're correct :+1:  
Been overthinking this, thanks.

**Comment by viceice** [comment]
i'e a working regex versioning

https://github.com/visualon/docker-buildpack/blob/main/.github/renovate.json#L48-L53

**Reply by j2L4e** [reply]
Thank you very much!
